src/snapshot/entity_checksum.rs

```diff
@@ -24,10 +24,7 @@ impl EntityChecksumPlugin {
 
         let result = ChecksumPart(hasher.finish() as u128);
 
-        trace!(
-            "Rollback Entities have checksum {:X}",
-            result.0
-        );
+        trace!("Rollback Entities have checksum {:X}", result.0);
```

Comment:
Thanks for doing this. I kept tripping this change while I was working on the other PRs and kept reverting it since it wasn't a part of my current work!